### PR TITLE
Issue #3035491 - Coding standards (part 2)

### PR DESCRIPTION
**Pull request by vever001**
These changes contain fixes for the DrupalPractice sniffs.

### CODE REVIEW

**Comment by vever001** [comment]
I'll fix the conflicts shortly

File: search_api_solr.api.php [hunk 1]
```diff
@@ -124,8 +124,7 @@ function hook_search_api_solr_converted_query_alter(\Solarium\Core\Query\QueryIn
     // the Solr index.
     $solr_field_names = $query->getIndex()->getServerInstance()->getBackend()->getSolrFieldNames($query->getIndex());
 
-    /** @var \Solarium\Component\EdisMax $edismax */
-    $edismax = $solarium_query->setQuery($solr_field_names['title'] . ':' . $solarium_query->getHelper()->escapePhrase('foo') . '^11.0');
```
Comment (mkalkbrenner) [inline search_api_solr.api.php:128]: This is an example. The return type should be covered. Instead removing, the better fix will be to add a line that uses $edismax.

**Review by mkalkbrenner — COMMENTED** [review]
Just on small adjustment required.

File: search_api_solr.api.php (hunk 1 above)
Comment (vever001) [reply search_api_solr.api.php:128]: You're right, I just tested this but `setQuery()` returns itself. 
In my test a `\Solarium\QueryType\Select\Query\Query` object which implements indeed `\Solarium\Core\Query\QueryInterface`
So how can $edismax be a `\Solarium\Component\EdisMax` object?

File: phpcs.xml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+  <rule ref="Drupal.WhiteSpace.ObjectOperatorIndent"/>
+  <rule ref="Drupal.WhiteSpace.ObjectOperatorSpacing"/>
+  <rule ref="Drupal.WhiteSpace.OpenBracketSpacing"/>
+  <rule ref="Drupal.WhiteSpace.OpenTagNewline"/>
+  <rule ref="Drupal.WhiteSpace.OperatorSpacing"/>
+  <rule ref="Drupal.WhiteSpace.ScopeClosingBrace"/>
+  <rule ref="Drupal.WhiteSpace.ScopeIndent"/>
+
+  <!-- Drupal Practice sniffs -->
+  <rule ref="DrupalPractice.Commenting.ExpectedException"/>
```
Comment (vever001) [inline phpcs.xml:90]: With this phpcs ruleset which is much more relaxed (only takes one sniff from the DrupalPractice ruleset), most of the fixes I did in commit a072c3ab72d495ac628023ea3edabab632409bf3 are not really required.
Let me know if you want me to revert those changes.

File: search_api_solr.api.php (hunk 1 above)
Comment (mkalkbrenner) [reply search_api_solr.api.php:128]: My fault. We changed a lot of things. It's better to keep your fix and to add another example.

**Pull request merged**